stdlib/2.7/sys.pyi

```diff
@@ -86,6 +86,9 @@ path_hooks = ...  # type: List[Any]
 path_importer_cache = ...  # type: Dict[str, Any]
 displayhook = ...  # type: Optional[Callable[[int], None]]
 excepthook = ...  # type: Optional[Callable[[type, BaseException, TracebackType], None]]
+exc_type = ...  # type: type
+exc_value = ...  # type: BaseException
```

Comment:
I have no idea -- I don't want to look it up in the CPython source any more than you do. :-)

That said these types look fine with me -- even though even when you do `raise IOError, 42`, `sys.exc_value` is still an IOError instance (it prints as `42` only because that's how the str() or an exception is defined).

In any case, the docs say these have been deprecated since Python 1.5! Do we really need them? (Well I guess the OP found a need...)